docs/sources/developers/plugins/create-a-grafana-plugin/extend-a-plugin/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Extend a Grafana plugin
+menuTitle: Extend a plugin
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - development
+  - extension
+  - documentation
+description: An index of how-to topics for extending or enhancing Grafana plugins.
+weight: 200
+---
+
+# Extend a Grafana plugin
+
+This section contains how-to topics for extending or enhancing Grafana plugins.
+
+- [Add annotations]({{< relref "./add-support-for-annotations.md" >}})
+- [Add anonymous usage reporting]({{< relref "./add-anonymous-usage-reporting.md" >}})
+- [Add authentication for a data source plugin]({{< relref "./add-authentication-for-data-source-plugins.md" >}})
+- [Add distributed tracing for backend plugins]({{< relref "./add-distributed-tracing-for-backend-plugins.md" >}})
+- [Add Explore queries]({{< relref "./add-support-for-explore-queries.md" >}})
+- [Add query editor help]({{< relref "./add-query-editor-help.md" >}})
+- [Add variables]({{< relref "./add-support-for-variables.md" >}})
+- [Automate development with CI](https://grafana.github.io/plugin-tools/docs/ci)
+- [Build a custom panel option editor]({{< relref "./custom-panel-option-editors.md" >}})
+- [Create nested plugins](https://grafana.github.io/plugin-tools/docs/nested-plugins)
+- [Extend configurations](https://grafana.github.io/plugin-tools/docs/advanced-configuration)
+- [Work with cross-plugin links]({{< relref "./cross-plugin-linking.md" >}})
+- [Work with data frames]({{< relref "./working-with-data-frames.md" >}})
```

Comment:
Work with DataFrames is pretty fundamental... I would move under Develop a plugin

Reply:
I've opened https://github.com/grafana/grafana/issues/70873 so we can make this move soon when we do additional moves. (There are a lot of links to this file that this would break if we did it now.)

Reply:
We're changing links and layout now, its quite awkward to keep doing this for users. The change would be easy as just search for refs to  /develop-a-plugin/working-with-dataframes and replace with /extend-a-plugin/working-with-dataframe

Reply:
Agreed. This is a good time for making the change.